spec/features/user_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.feature 'Users', type: :feature, feat: true do
+  before(:all) do
+    @user = User.create(name: 'Jann Doe', bio: 'I am John Doe', email: 'rand_email')
+  end
+
+  after(:all) do
+    @user.destroy
+  end
+
+  it 'displays all users' do
+    visit users_path
+    expect(page).to have_content('Users')
+  end
+
+  it 'displays a user' do
+    # user = User.create(name: 'John Doe', bio: 'I am John Doe')
+    # puts user.id
+    visit user_path(@user)
+    expect(page).to have_content('John Doe')
+  end
+
+  it 'displays a user edit page' do
+    # user = User.create(name: 'John Doe', bio: 'I am John Doe')
+    visit edit_user_path(@user)
+    expect(page).to have_content('Edit User')
+  end
+end
```

Comment:
#### Please ensure featured tests for `User` show page is implemented and includes the following tests to meet all the project requirements:

* Tests that assert you can see the user's profile picture.
* Tests that asserts you can see the user's username.
* Tests that asserts you can see the number of posts the user has written.
* Tests that assert you can see the user's bio.
* Tests that assert you can see the user's first 3 posts.
* Tests that assert you can see a button that lets me view all of a user's posts.
* Tests that redirect to the post's show page when you click on a user's post.
* Tests that redirect to the user's post's index page when you click on a **see all posts**.

Reply:
#### Please ensure featured tests for `User` index page is implemented and includes the following tests to meet all the project requirement:

* Tests that assert you can see the username of all other users.
* Tests that assert you can see the profile picture for each user.
* Tests that assert you can see the number of posts each user has written.
* Tests that redirect to the user's show page when you click on a user.